lightstep/resource_alert_test.go

```diff
@@ -680,22 +680,93 @@ resource "lightstep_alert" "errors" {
 `
 
 	compositeConditionConfig := `
+	resource "lightstep_slack_destination" "slack" {
```

Comment:
the diff is a little weird here -- I didn't change the existing test alerts, just added a few new ones that are largely identical (so github got confused)